qa/t/test_bug_edit.t

```diff
@@ -40,7 +40,7 @@ check_page_load($sel, q{http://HOSTNAME/editgroups.cgi});
 $sel->title_is("Edit Groups");
 $sel->click_ok("link=Master");
 check_page_load($sel,
-  q{http://HOSTNAME/editgroups.cgi?action=changeform&group=29});
+  q{http://HOSTNAME/editgroups.cgi?action=changeform&group=30});
```

Comment:
What does this do?

Reply:
Tech debt. I need to make the test script smarter in how it gets the group id for the Master group. This step might actually be able to removed. But what it does it assumes that the last group that was added is the id 29 (now 30) and since I added new permission group in this PR, it also increased the number needed in the test script by 1 :(